aten/src/ATen/CMakeLists.txt

```diff
@@ -232,6 +232,14 @@ if(USE_CUDA AND (USE_FLASH_ATTENTION OR USE_MEM_EFF_ATTENTION))
   )
 
   set_target_properties(flash_attention PROPERTIES POSITION_INDEPENDENT_CODE ON)
+
+  if(DEFINED ENV{FA_MAX_JOBS} AND CMAKE_GENERATOR STREQUAL "Ninja")
```

Comment:
Should we throw a warning here if FA_MAX_JOBS is set but generator is not Ninja? Seems like a footgun without some logging.